More Train Updates

Dunrab opened the pull request:
# Description

A lot of fixes for Train:

- fixed missing HV wire going from HOP/Dorms car to the car above causing power issues
- placed missing APC in reporters office
- placed missing APC in jani closet
- placed missing APC in botany
- removed the station beacon for the jani closet that was left in the bar
- removed the dirty tiles from the front of hops office that could not be cleaned
- connected some disconnected disposal units
- turned the north part of the bar into an arcade since the station lacked one (prize counter included)
- added the gavel block and trial timer to the court house since it was missing
- fixed the power problem in medicals treatment room where the LV wasn't brought up enough to power it
- replaced the paramedlockers that were empty with paramedlockerfilled so they will have gear, kept the hardsuit lockers seperate
- replaced the airlock assembly on the carp shuttle above security with actual docking airlocks so it would be easier to repair and turn into a flyable shuttle
- replaced the captain spare id that was manually placed in the caps outer office with a spareidcabinet thats above the meeting table
- detectives office now has the right station beacon
- Corpsman room has a station beacon now
- fixed a lot of missing air alarms/air sensors
- fixed a lot of missing and or misplaced station beacons
- added holopads to each department
- added station ai (did not open a job slot for, will wait till we are ready)
- botany starts with a chem master and beaker now
- Engineering starts with a plasma tank dispenser so they can actually run the singulo if they wanted to
- CE's gets extra RCD ammo in their office

---

<details><summary><h1>Map Render</h1></summary>
<p>

<img width="2560" height="13504" alt="Train-0" src="https://github.com/user-attachments/assets/a2d34d9b-f06a-42b4-9c05-0e158773da33" />


</p>
</details>

---

# Changelog
:cl:
- tweak: More tweaks to train to fix some of power issues, air alarm issues and other minor things like station beacons.

Dunrab committed:
more train updates

see pr for full changelog is a lot

Dunrab committed:
holopads and ai

placed holopads and ai jobspawn

Mnemotechnician reviewed (approved):
Looks fine, I see no issues on the map render besides the diagonal wall render bug

M3739 reviewed (approved):
Same case as with #1454, I won't merge this until the deployment PR is made. which may be either tonight, or tomorrow.

By then, I intend to merge it.

Dunrab committed:
small fixes

added rcd ammo to CE's office
added a plasma tank dispesner to engineering so engi can actually run singulo
added missing APC to the arcade so it will have power

Dunrab committed:
few more small fixes

botany has a chem master and starts with a beaker

Dunrab committed:
add clerk locker to clerk room

adds a clerk locker to clerk room in the court house, some how i missed this

M3739 commented:
Merging in 1 hour.
